src/Compilers/CSharp/Test/Syntax/Syntax/SyntaxEquivalenceTests.cs

```diff
@@ -437,5 +437,15 @@ public void TestUpdateInterpolatedString()
             VerifyEquivalent(tree1, tree2, topLevel: true);
             VerifyNotEquivalent(tree1, tree2, topLevel: false);
         }
+
+        [Fact(DisplayName = "https://github.com/dotnet/roslyn/issues/7380")]
+        public void TestExpressionBodiedMethod()
```

Comment:
Since the arrow clause is not part of the API (when `topLevel:true`), we don't care about the errors. When `topLevel:false`, we retain the old behavior.